subsys/bluetooth/mesh/time_srv.c

```diff
@@ -243,6 +265,11 @@ static int handle_time_set(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *
 		srv->time_update_cb(srv, ctx, BT_MESH_TIME_SRV_SET_UPDATE);
 	}
 
+	/* The time server shouldn't send out statuses more often than every 30 seconds. */
+	if ((srv->data.timestamp + RESTRICTION_TIME) < srv->data.sync.uptime) {
+		(void)bt_mesh_time_srv_time_status_send(srv, NULL);
+	}
+
```

Comment:
I think this change is not needed. If 'client' is manually setting the time, then we do want client to be notified with status as soon as possible. Since statuses can be lost due to collisions, client should be able to retry again quickly.

Reply:
It is immediately notified. Please take a look at the next code line.